webknossos/webknossos/geometry/bounding_box.py

```diff
@@ -101,7 +108,7 @@ def from_auto(
             else:
                 return BoundingBox.from_csv(obj)
         elif isinstance(obj, dict):
-            return BoundingBox.from_wkw(obj)
+            return BoundingBox.from_wkw_dict(obj)
```

Comment:
We might also disambiguate wkw_dicts and config dicts:
```suggestion
            if "size" in obj:
                return BoundingBox.from_config_dict(obj)
            else:
                return BoundingBox.from_wkw_dict(obj)
```